src/Serilog.Enrichers.Thread/project.json

```diff
@@ -1,21 +1,28 @@
 ﻿{
-  "version": "2.0.0-beta-*",
+  "version": "2.0.0-*",
```

Comment:
With the current build setup, this will result in `2.0.0-123` type version numbers from master. In the other projects we've just hard-coded `2.0.0` to get the release build out, but if we want to keep this style then the `DNX_BUILD_VERSION` variable needs to be empty on master, and populated with something like `beta-123` on other branches.